scripts/manage_tools.sh

```diff
@@ -43,7 +43,11 @@ add_plugins() {
 	for plugin in "${wanted_plugins[@]}"; do
 		if ! array_contains "${plugin}" "${added_plugins[@]}"; then
 			printf 'Adding asdf plugin: "%s"\n' "${plugin}"
-			asdf plugin add "${plugin}"
+			if [[ $plugin == "circleci" ]]; then
```

Comment:
Why are we we installing the plugin differently here?

Reply:
I think we should make this as a mapping somewhere, instead of doing if else dance in the function - either create a hashmap in manage_tools.sh, or add repository needed to install given to `.tool-versions` if possible.

@sbienkow-ninja any idea how we could more easily support asdf tools that require installation of plugins from specific repositories?